wsd/TileDesc.hpp

```diff
@@ -215,9 +215,9 @@ class TileDesc final
         TileWireId wireId = 0;
         for (std::size_t i = 0; i < tokens.size(); ++i)
         {
-            if (LOOLProtocol::getTokenUInt32(tokens[i], "oldwid", oldWireId))
+            if (tokens.getUInt32(i, "oldwid", oldWireId))
                 ;
-            else if (LOOLProtocol::getTokenUInt32(tokens[i], "wid", wireId))
+            else if (tokens.getUInt32(i, "wid", wireId))
                 ;
             else
             {
```

Comment:
Awesome - but I think you missed the hottest call in the line below (?) =)

Reply:
Yes, I did. I've now replaced LOOLProtocol::parseNameIntegerPair() with StringVector::getNameIntegerPair() in a follow-up commit that attempts to avoid copying in that case as well.